src/System/Wlog/LoggerConfig.hs

```diff
@@ -122,8 +123,9 @@ instance Monoid LoggerTree where
 
 instance ToJSON HandlerWrap
 instance FromJSON HandlerWrap where
+    -- we use 'normalise' so that Unix paths would work on Windows
     parseJSON = withObject "handler wrap" $ \o -> do
-        (_hwFilePath :: FilePath) <- o .: "file"
+        (_hwFilePath :: FilePath) <- normalise <$> o .: "file"
```

Comment:
http://hackage.haskell.org/package/filepath-1.4.1.2/docs/System-FilePath-Windows.html#v:normalise says that it converts `/` to `pathSeparator`, so this should correctly convert relative Unix paths (such as `pub/node.pub`) to Windows paths on Windows